_extensions/posit-docs/theme-dark.scss

```diff
@@ -3,40 +3,10 @@
 // import font
 @import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap");
```

Comment:
Is it worth sharing the font definition as well?

Reply:
I was wondering about that. Maybe a shared imports that includes colors and fonts and ... ? Although unless there's anything else to add, maybe we just leave it for now.

Reply:
Yeah. I think that we can leave it for now and revisit?